web/packages/build/.eslintrc.js

```diff
@@ -67,8 +67,18 @@ module.exports = {
     },
   ],
   rules: {
-    'no-unused-vars': 'off', // disabled to allow the typescript one to take over and avoid errors in reporting
-    '@typescript-eslint/no-unused-vars': ['error'],
+    // Turn on the no-unused-imports rule. As it works by wrapping
+    // @typescript-eslint/no-unused-vars, we need to turn this one off, and
+    // instead use the wrapped one.
+    '@typescript-eslint/no-unused-vars': 'off',
+    'no-unused-vars': 'off',
+    'unused-imports/no-unused-imports': 'error',
+    'unused-imports/no-unused-vars': [
+      'error',
+      {
+        ignoreRestSiblings: true,
```

Comment:
What's the reason behind this being different than on master? I don't think the option itself is a bad option to pick, just curious about it being different than on master. I recently ran into a scenario where having this set to true would've been useful.